Codeinwp/optimole-wp
# fix: error handling on plugin registeration

### All Submissions:

 

### Changes proposed in this Pull Request:
This PR adds email validation and fixes already registered email error not showing up. For the invalid email, it triggers when you click outside the email field and the connect button also stays disabled if the email isn't valid.
 

Closes https://github.com/Codeinwp/optimole-wp/issues/456, https://github.com/Codeinwp/optimole-service/issues/1037.

### How to test the changes in this Pull Request:

1. Make sure you can't connect with an invalid email address, such as hello#gmail.com and etc.
2. Make sure if you are connecting with an email already registered, you get a notice for email already registered.

### Other information:
